.github/workflows/integration-test.yml

```diff
@@ -1,7 +1,10 @@
 on:
+    push:
```

Comment:
Are the changes in this file intended to be checked in?

Reply:
Yes, integration test is still broken, and I want to address it in this PR as well.